sdk/python/feast/infra/utils/snowflake/snowpark/snowflake_udfs.py

```diff
@@ -24,6 +25,8 @@
 # ValueType.BYTES = 1
 @vectorized(input=pandas.DataFrame)
 def feast_snowflake_binary_to_bytes_proto(df):
+    sys._xoptions["snowflake_partner_attribution"].append("feast")
```

Comment:
Unrelated to this PR, but I'm curious how the python client serializes these UDFs - is there any documentation or a place in the code you could point me to?